src/main/java/org/symphonyoss/symphony/messageml/elements/RegexElement.java

```diff
@@ -92,6 +92,10 @@ default void validateRegex() throws InvalidInputException {
       presentationAttrs.put(PRESENTATIONML_PATTERN_ERROR_MESSAGE_ATTR, getAttribute(PATTERN_ERROR_MESSAGE_ATTR));
     }
 
+    if (getAttribute(PRESENTATIONML_PATTERN_ERROR_MESSAGE_ATTR) != null) {
+      presentationAttrs.put(PRESENTATIONML_PATTERN_ERROR_MESSAGE_ATTR, getAttribute(PRESENTATIONML_PATTERN_ERROR_MESSAGE_ATTR));
```

Comment:
don't we have similar issues with all the existing elements that are using _attribute_ in MML and _data-attribute_ in PML?

Reply:
Yes had a lot of similar issues actually, most of them have already been fixed part of https://github.com/finos/messageml-utils/pull/315. This particular use-case was missing from [our set of examples](https://github.com/finos/messageml-utils/tree/master/src/test/resources/examples).